### PR TITLE
CXSPA-1291: The dot loading-indicator in both csagent-login-form component and cx-customer-selection component is replaced with the newly extracted component.

### PR DESCRIPTION
**Pull request by luoyifeng11**
Jira: https://jira.tools.sap/browse/CXSPA-1291

### CODE REVIEW

**Comment by ercultimate** [comment]
Code-wise, this is excellent.  To complete the DoD for a breaking change, we must also document any changes in the relevant migration document.  Please see this wiki document https://wiki.one.int.sap/wiki/display/spar/Guideline%3A+Breaking+Change+Handling+in+Major+Version+Releases for more details.

**Review by ercultimate — CHANGES_REQUESTED** [review]
Code-wise, this is excellent, we are just missing some documentation required by the DoD.

The DoD says that for breaking changes we must also document any changes in the relevant migration document. 
Please see this wiki document https://wiki.one.int.sap/wiki/display/spar/Guideline%3A+Breaking+Change+Handling+in+Major+Version+Releases for more details on what documents need entries for these changes.

**Comment by luoyifeng11** [comment]
>

**Comment by luoyifeng11** [comment]
> The DoD says that for breaking changes we must also document any changes in the relevant migration document. 

The styling changes have been added in the migration document.

File: feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ describe('CustomerSelectionComponent', () => {
     fixture.detectChanges();
 
     // TODO: (CXSPA-1291) Change div.spinner to use cx-dot-spinner component
```
Comment (ercultimate) [inline feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts]:
```suggestion
```

File: feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html [hunk 2]
```diff
@@ -32,13 +32,8 @@
   </button>
 </form>
 
-<div
+<cx-dot-spinner
```
Comment (ercultimate) [inline feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html:35]: I just realized that this HTML change needs documented as well.

**Review by ercultimate — CHANGES_REQUESTED** [review]
Style documentation looks good to me.  We just need to add the documentation for the HTML change of `div.spinner` -> `cx-dot-spinner`, then this will be ready for merging.

File: feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts (hunk 1 above)
Comment (luoyifeng11) [reply feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts]: Done

File: feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html (hunk 2 above)
Comment (luoyifeng11) [reply feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html:35]: Done

**Review by ercultimate — DISMISSED** [review]

**Review by kimhw0630 — DISMISSED** [review]

**Review by ercultimate — APPROVED** [review]

**Pull request merged**